src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int firstSide;
+    private int secondSide;
+
+    public Rectangle(int firstSide, int secondSide, Color color) {
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+        super.color = color;

```

Comment:
Creating a figure, don't pass expressions in the constructor. It's better to create separate variables and pass them on for better code readability. Additionally, you should call the parent constructor using `super(color)` instead of directly assigning the color to the `super.color` field.